data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -921,8 +926,30 @@ def build_filing_json_meta_data(raw_filing_type: str, filing_type: str, filing_s
             'stateChange': state_change,
             'nameChange': name_change,
         }
-
-    if filing_type == 'annualReport':
+        
+        if accession_num := filing_data.get('accession_num'):
+            meta_data['conversion']['accessionNumber'] = accession_num
+
+    if filing_type == 'conversionLedger':
+        filing_cars_data = next(
+            (item for item in data['cars_data'] if item.get('event_id') == filing_data['e_event_id']), None
+        )
+        if filing_cars_data:
+            meta_data['conversion'] = {
```

Comment:
@argush3 any thoughts on adding CARS data inside conversion (even for conversionLedger). This way it would be easier to use single logic for both

Reply:
sounds good to me